src/lib/parts.ts

```diff
@@ -196,6 +196,9 @@ export class NodePart implements Part {
       this._commitNode(value);
     } else if (Array.isArray(value) || value[Symbol.iterator]) {
       this._commitIterable(value);
+    } else if (value === nothing) {
+      this.value = undefined;
```

Comment:
keeping `nothing` as the value would make it cleat that the part is empty

Reply:
I opted to set to undefined because that's the initial state, which seemed reasonable to me. 

If we set the value to `nothing` we can test against it, but I think calling `clear` on an empty NodePart is almost free anyway, so I'd rather just always clear and reduce bytesize.

Now that I think about it, `value` probably compresses better than `undefined`, and it doesn't really make any difference what we use as long as we override the previous value, so I'll change it to save some more bytes.